src/Fmt_ast.ml

```diff
@@ -1745,8 +1780,7 @@ and fmt_expression c ?(box = true) ?epi ?eol ?parens ?ext
             (parens || (not (List.is_empty pexp_attributes)))
             "(" ")"
             ( hvbox 2
-                ( fmt_module c keyword name xargs (Some xbody) true xmty
-                    (List.append pmod_attributes pmod.pmod_attributes)
```

Comment:
Maybe I have lost the plot, but why are these attributes no longer needed?

Reply:
First, `pmod_attributes` and `pmod.pmod_attributes` are the same value.
Second, attributes are already formatted by `fmt_module -> fmt_module_expr c`.
Note that, the change is part of a in a "Fix" commit on its own.

Reply:
Ah I see, ok good, thanks.